src/core/style/themes.scss

```diff
@@ -39,4 +39,10 @@ html, body {
   &#{$checkedSelector} ._md-icon:after {
     border-color: '{{primary-contrast-0.87}}';
   }
+
+  & .md-indeterminate[disabled] {
```

Comment:
Indeterminate isn't used anywhere else, only on progress and this seems not related, 
Also don't you think this class should be private?